Core/CMP API Specification.md

```diff
@@ -1042,7 +926,7 @@ The sent message shall follow the form outlined below. The command, parameter an
   __gppCall: {
     command: "command",
     parameter: “parameter”,
```

Comment:
can you correct the " ? we have three different quotes here (normal, cursive and single quotes) - lets stick to one ;-)

Reply:
updated